### ISSUE TITLE
Angular 15 / Webpack Issue

### ISSUE THREAD

**Issue by inloox-dev** [NoelOConnell/quill-image-uploader#74]
The following line requires a custom Webpack configuration:

https://github.com/NoelOConnell/quill-image-uploader/blob/b535852f48d3bcf9390e59f31582a2f4e3858a6a/src/quill.imageUploader.js#L3

This is the output of an Angular 15 production build:

```
> ./node_modules/quill-image-uploader/src/quill.imageUploader.css:1:0 - Error: Module parse failed: Unexpected token (1:0)
> You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> > .image-uploading {
> | position: relative;
> | display: inline-block;
```

We would like to discuss if there is a better solution that does not require a custom Webpack loader / configuration.

Looking forward to hearing your thoughts. Thank you!

**Comment by atremel** [comment]
Hi,

It really poses a problem to us. According to this importing CSS in Javascript is not considered to be standard:

https://github.com/angular/angular-cli/issues/23273#issuecomment-1173861005

> Importing a Sass/CSS file inside your TypeScript/JavaScript file like an ECMA module
> is no not a standard functionality and as such this part is not supported.

It would be good to stick to the standard IMHO.

**Comment by jgillick** [comment]
I've created a fix for this in PR: https://github.com/NoelOConnell/quill-image-uploader/pull/87

**Comment by NoelOConnell** [comment]
Hopefully this is fixed now with package version `1.3.0` now thanks to @jgillick for the PR.
